contracts/Unibright.sol

```diff
@@ -142,28 +142,28 @@ contract UBTSplitter is Context, Ownable {
      * percentage of the total shares and their previous withdrawals. `token` must be the address of an IERC20
      * contract.
      */
-    function release(address revenueAddress) public virtual {
+    function release() public virtual {
         require(
-            shares[revenueAddress] > 0,
-            "revenueAddress has no shares"
+            shares[msg.sender] > 0,
```

Comment:
is it expected that the only one who can trigger a release is the shares holder? What happens in a scenario where the original payee has lost access to account behind the shares? Do we want to introduce a mechanism where the council could still release them funds?

Reply:
with current implementation yes it is expected, might be improvement in future, also task candidate?